src/shims/unix/linux/eventfd.rs

```diff
@@ -9,8 +10,8 @@ use crate::{concurrency::VClock, *};
 
 use self::shims::unix::fd::FileDescriptor;
 
-/// Minimum size of u8 array to hold u64 value.
-const U64_MIN_ARRAY_SIZE: usize = 8;
+// Size of u8 array to hold u64.
+const U64_ARRAY_SIZE: usize = mem::size_of::<u64>();
```

Comment:
Is it ok to leave this as const? ``bytes.first_chunk_mut::<U64_ARRAY_SIZE>()`` can only take in const for ``U64_ARRAY_SIZE``.

Reply:
Yeah a const is fine.